_layouts/page.html

```diff
@@ -4,5 +4,6 @@
 
 <div class="page">
   <h1 class="page-title">{{ page.title }}</h1>
-  {{ content }}
+  <a href="{{ site.baseurl }}{{ page.url }}"></a>
```

Comment:
I am not questioning the href, but the HTML element. A hyperlink. It is empty. Doesn't serve any purpose on the browser, unless I am missing something.